contents/handbook/product/product-stages.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Product stages
+sidebar: Handbook
+showTitle: true
+---
+
+We have multiple teams working on multiple products at various stages, it's essential to have a clear understanding of our focus and priorities. This guide helps answer crucial questions related to product development, launches, pricing, and growth. Please note that this is a rough guide, and you should adapt it to your teams and individual products.
+
+It should help address confusion around when to focus on feature parity vs innovative new major features. How much do we need to care about stability vs it being fine to move fast and break things. Should we keep investing in the smaller features or is doing the first 80% fine.
+
+**Caveat:** this guide mainly applies to building products where the category already exists. Defining a new category often requires different ordering/thinking.
+
+## The spectrum of product development
+
+Our products fall under three main categories within the product development spectrum:
+
+1. Minimum Viable Product (MVP)
+2. Product-Market Fit (PMF)
+3. Standalone Best-in-Class
```

Comment:
```suggestion
3. Best-in-Class
```

This will often come from being _integrated_ so the word "standalone" I think is confusing